vernac/vernacexpr.ml

```diff
@@ -130,7 +130,7 @@ type definition_expr =
   | DefineBody of local_binder_expr list * Genredexpr.raw_red_expr option * constr_expr
       * constr_expr option
 
-type decl_notation = lstring * constr_expr * scope_name option
+type decl_notation = lstring * constr_expr * bool * scope_name option
```

Comment:
Probably the right time to turn this into a record.

Reply:
Yes, this is what I thought too... Maybe tonight or tomorrow (or even later if not urgent)?

Reply:
I experimenting using a record. It is much more verbose, but certainly also more self-explanatory. Any suggestions in passing on how formatting records?

Reply:
You mean, the type declaration? I subscribe to the 
```
type foo = {
  foo_bla : Tbla;
  foo_bar : Tbar;
  ...
}
```
syntax, but I am not sure that's what your asking.

Reply:
> but I am not sure that's what your asking.

Yes, that was part of my question. In practice, I used:
```
type decl_notation =
  { decl_ntn_string : lstring
  ; decl_ntn_interp : constr_expr
  ; decl_ntn_only_parsing : bool
  ; decl_ntn_scope : scope_name option
  }
```
because that's the style which was already used nearby in `vernacexpr.ml` (even if other part of `vernacexpr.ml` use the style that you're mentioning.

On the other side, in `metasyntax.ml`, I used the style:
```
    let open Vernacexpr in
    let
      { decl_ntn_string = {CAst.loc;v=ntn};
        decl_ntn_interp = c;
        decl_ntn_only_parsing = onlyparsing;
        decl_ntn_scope = scopt } = decl_ntn in
```